manifest_dev.yml

```diff
@@ -5,6 +5,9 @@ applications:
 - name: api
   path: .
   host: fec-dev-api
+  env:
+    NEW_RELIC_APP_NAME: OpenFEC (development)
```

Comment:
Can we change this (and the others) to "OpenFEC API (env)"? Since we'll also be reporting from "OpenFEC Web (env)".